### ISSUE TITLE
Make Coupling Metrics more reliable

### ISSUE THREAD

**Issue by ResistantBear** [MaibornWolff/metric-gardener#45]
Inside the codebase for the coupling metrics, there are queries that might not cover any case that the grammar of the language allows for (e.g. the ones touched by #43). This should be fixed.
Additionally, more detailed tests for the coupling metrics would be good.

**Comment by ce-bo** [comment]
Good point. We would need to identify existing issues first. However, I would like to focus more on working/improving basic file metrics first.
